### PR TITLE
css-grid-classes | Eniola Johnson

### PR DESCRIPTION
**Pull request by eniolajohnson**
image

### CODE REVIEW

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great initial implementation! Please update with the correct use of rows and column containers when time allows!

**Comment by eniolajohnson** [comment]
image
Edited to reflect feedback.

**Comment by eniolajohnson** [comment]
Inappropriate lines of code fixed.
image

File: css-grid-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <title>CSS Grid Classes</title>
+</head>
+<body>
+    <div class="container">
+      <div class="row">
+        <h1 class="column_full">Choose your character!</h1>
+      </div>
+
+      <div class="row">
+        <img class="column_half" src="./images/mario.png" alt="">
```
Comment (Zhiwutian) [inline css-grid-classes/index.html:16]: make sure to wrap these images and all other images for this exercise in a column container. it is a best practice to put the column classes on the container, and then to set the width of the images in the containers to be 100% of the width of the containers. You don't want to put the column class directly on the img element.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one more small update based on comments!

File: css-grid-classes/index.html (hunk 1 above)
Comment (eniolajohnson) [reply css-grid-classes/index.html]: Aye aye, Captain! Here is the edited version based on your feedback.

image
image

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**